### PR TITLE
enh: add new run type for Use tab

### PR DESCRIPTION
**Pull request by fontanierh**
Do we want this @spolu  ?

### CODE REVIEW

File: core/src/run.rs [hunk 1]
```diff
@@ -134,13 +134,15 @@ impl RunStatus {
...
     Local,
+    Execute,
 }
 
 impl ToString for RunType {
     fn to_string(&self) -> String {
         match self {
             RunType::Deploy => "deploy".to_string(),
             RunType::Local => "local".to_string(),
+            RunType::Execute => "execute".to_string(),
```
Comment (fontanierh) [inline core/src/run.rs:145]: Went with "execute", felt like "run" was ambiguous and "use" too much tied to the frontend

**Comment by fontanierh** [comment]
cc @gabhubert as you mentioned it on my Use tab PR:

Screenshot 2023-03-24 at 14 56 44

This adds to the `Logs` tab a third sub-tab for the runs that are done from the use tab. Alternatively, we could keep them in the "local" sub-tab, but find a way to visually separate the runs done from `specification` tab from the ones done from the `use` tab

File: front/pages/[user]/a/[sId]/runs/index.jsx [hunk 2]
```diff
@@ -16,6 +16,7 @@ const { URL, GA_TRACKING_ID = null } = process.env;
 const tabs = [
   { name: "Local", runType: "local" },
   { name: "API", runType: "deploy" },
+  { name: "Use", runType: "execute" },
```
[the remark this answers is not in the document]
Comment (spolu) [reply front/pages/[user]/a/[sId]/runs/index.jsx:19]: Yes I think we want it in the middle.

I would rename Local to "Design" and "Use" to "Execute" (that's the name of the button in the "Use" tab already)

[the remark this answers is not in the document]
Comment (fontanierh) [reply front/pages/[user]/a/[sId]/runs/index.jsx:19]: Sounds good

File: core/src/run.rs (hunk 1 above)
Comment (spolu) [reply core/src/run.rs:145]: Yes that looks good to me. Maybe we'll migrate "Local" to "Design" in the future.

**Review by spolu — APPROVED** [review]

**Comment by fontanierh** [comment]
> Woot CI running :)

It's pretty slow (~6m) because it builds all dependencies. I'll try to look for a way to improve that !

**Pull request merged**